ui/tests/integration/components/secure-variable-form-test.js

```diff
@@ -144,4 +144,30 @@ module('Integration | Component | secure-variable-form', function (hooks) {
       );
     });
   });
+
+  test('Prevent editing path input on existing variables', async function (assert) {
+    assert.expect(3);
+
+    const variable = await this.server.create('variable', {
+      name: 'foo',
+      namespace: 'bar',
+      path: '/baz/bat',
+      keyValues: [{ key: '', value: '' }],
+    });
+    variable.isNew = false;
```

Comment:
`question`:  Are we unable to write (below) because we haven't added this property to the model yet?

```
    const variable = await this.server.create('variable', {
      name: 'foo',
      namespace: 'bar',
      path: '/baz/bat',
      keyValues: [{ key: '', value: '' }],
      isNew: false,
    });
```

Reply:
Turns out it's slightly more nuanced than this; Ember Mirage's implementation of isNew returns as a function (https://www.ember-cli-mirage.com/versions/v0.3.x/models/) on this.server.create(), so we set it afterwards to override the function property.

Reply:
(it's necessary to do this at all because mirage models are not themselves ember models)